config/prod/jhu-biobank-nf-project.yaml

```diff
@@ -0,0 +1,24 @@
+stack_name: jhu-biobank-nf-project
```

Comment:
Nit:
Would prefer that the order of entries in the project yaml files follows the standard order.
Starting with template_path, stack_name, etc. Tags at bottom.